includes/opdat_4.3.php

```
@@ -29,690 +29,9 @@
-		#####
-		$qtxt = "SELECT column_name
-				FROM information_schema.columns
-				WHERE table_name = 'adresser'
-				AND column_name = 'fax'";
-
-		if (db_fetch_array(db_select($qtxt, __FILE__ . " linje " . __LINE__))) {
-			$qtxt = "ALTER TABLE adresser RENAME COLUMN fax TO mobile";
-			db_modify($qtxt, __FILE__ . " linje " . __LINE__);
-		}
-	});
+	function opdat_4_3($majorNo, $subNo, $fixNo){
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$majorNo'. (undefined)

(UnusedFormalParameter)